ansible/roles/csm.sbps.dns_srv_records/files/sbps_dns_srv_records.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
+
+#set -x
+
```

Comment:
I don't see any checking whether the commands being run in this script pass or fail. I would recommend either using the following setting (to cause the script to automatically fail if any command fails), or to add code to check for command failures and handle them.

```suggestion
set -euo pipefail

```